helm/ph-ee-fin-back-end/config/application-bb.properties

```diff
@@ -0,0 +1,13 @@
+store.local.interop.host=https://rhino.mifos.g2pconnect.io
+store.local.customer.host=https://rhino.mifos.g2pconnect.io
+zeebe.broker.contactpoint=zeebe-zeebe-gateway:26500
```

Comment:
are we still planning to use this with zeebe-zeebe here?

Reply:
yes, It will be same for safaricom